apis/policies/v1/clusteradmissionpolicy_webhook.go

```diff
@@ -84,7 +85,42 @@ func (r *ClusterAdmissionPolicy) ValidateUpdate(old runtime.Object) error {
 	return validatePolicyUpdate(oldPolicy, r)
 }
 
+func validatePolicyCreate(policy Policy) error {
+	return validateRulesField(policy)
+}
+
+// Validates the spec.Rules field for non-empty, webhook-generable rules
+func validateRulesField(policy Policy) error {
+	errs := field.ErrorList{}
+
+	if len(policy.GetRules()) != 0 {
```

Comment:
we should fail if `len(policy.GetRules())` is 0. Is that case posible?

Reply:
Updated!